v2/tools/generator/internal/astmodel/package_reference_set.go

```diff
@@ -56,29 +56,15 @@ func (set *PackageReferenceSet) Contains(ref PackageReference) bool {
 	return ok
 }
 
-// AsSlice returns a slice containing all the imports
-func (set *PackageReferenceSet) AsSlice() []PackageReference {
-	result := make([]PackageReference, 0, len(set.references))
-	for ref := range set.references {
-		result = append(result, ref)
+// All returns an iterator over all the references in the set
+func (set *PackageReferenceSet) All() iter.Seq[PackageReference] {
+	return func(yield func(ref PackageReference) bool) {
+		for ref := range set.references {
+			if !yield(ref) {
+				break
+			}
+		}
 	}
-
-	return result
-}
-
-// AsSortedSlice return a sorted slice containing all the references
-// less specifies how to order the imports
-func (set *PackageReferenceSet) AsSortedSlice(
```

Comment:
Was this function not actually used?

Interesting.

Reply:
I think we were using it at one point, but then we improved the way we generated imports and the order was no longer relevant.